client/package.json

```diff
@@ -8,6 +8,7 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
```

Comment:
We should be telling linter to be searching the `./src` directory instead of also `assets/**` and `public/**`